xml/Makefile.am

```diff
@@ -31,7 +31,7 @@ RNGdir			= $(dtddir)
 
 # Sorted list of available numeric RNG versions,
 # extracted from filenames like NAME-MAJOR[.MINOR][.MINOR-MINOR].rng
-RNG_numeric_versions    = $(shell ls -1 *.rng \
+RNG_numeric_versions    = $(shell echo $(wildcard $(top_srcdir)/xml/*.rng) \
```

Comment:
The Travis errors show problems with the XML schemas, so I suspect this wildcard isn't a drop-in replacement. You probably need to strip off the leading path in the sed.